test/Mono.Linker.Tests.Cases/Reflection/MembersUsedViaReflection.cs

```diff
@@ -313,7 +319,9 @@ private void PrivateMethod ()
 			public static class PublicNestedType
 			{
 				// PublicNestedType should be kept but linker won't mark anything besides the declaration
```

Comment:
Nit: We should delete these comments in our testing given that now the link will mark more than the declaration